package.json

```diff
@@ -94,45 +94,46 @@
     "zone.js": "0.11.5"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "13.3.5",
-    "@angular/cli": "13.3.5",
-    "@angular/compiler-cli": "13.3.6",
+    "@angular-devkit/build-angular": "14.0.0",
+    "@angular/cli": "14.0.0",
+    "@angular/compiler-cli": "14.0.0",
     "@bahmutov/add-typescript-to-cypress": "2.1.2",
     "@cypress/webpack-preprocessor": "5.11.1",
-    "@jest/globals": "28.1.0",
-    "@types/jest": "27.5.0",
+    "@jest/globals": "28.1.1",
+    "@types/jest": "28.1.1",
     "@types/js-yaml": "4.0.5",
     "@types/lodash": "4.14.182",
-    "@types/node": "17.0.36",
-    "@typescript-eslint/eslint-plugin": "5.22.0",
-    "@typescript-eslint/parser": "5.22.0",
+    "@types/node": "17.0.40",
+    "@typescript-eslint/eslint-plugin": "5.27.1",
+    "@typescript-eslint/parser": "5.27.1",
     "browserlist": "1.0.1",
+    "btoa": "1.2.1",
```

Comment:
Is this required by some other dependency/package?

Reply:
Yes, with the changes it becomes required.